safe-global/safe-wallet-web
# feat: Add SwapTokens component

## What it solves

Adds a `SwapTokens` component

## How to test it

1. Open the storybook and observe the new component
2. Make sure the props make sense

## Screenshots

<img width="1036" alt="Screenshot 2024-04-16 at 11 53 29" src="https://github.com/safe-global/safe-wallet-web/assets/5880855/c169f101-e58e-46a8-b396-fd6240d1bcf7">


## Checklist
